libraries/SITL/SITL.h

```diff
@@ -342,6 +342,8 @@ class SITL {
 
     uint16_t irlock_port;
 
+    double start_time_UTC;
```

Comment:
This is a rather unexpected type!

It should probably be `time_t`.

Reply:
Thanks, fixed